src/main/java/org/jboss/forge/arquillian/command/TestSetUp.java

```diff
@@ -0,0 +1,15 @@
+package org.jboss.forge.arquillian.command;
+
+
+import org.jboss.forge.roaster.model.source.JavaClassSource;
+import org.jboss.forge.roaster.model.source.MethodSource;
+
+public interface TestSetUp {
+
+    default void removeTestMethod(String methodName, JavaClassSource test) {
```

Comment:
I'm not sure if we really need a common, default behaviour for all the `TestSetup` implementations. It's very much dependent on the given case and maybe it would be better to move this logic to a component offering functionality of test removal and just use it in concrete implementations when necessary?

In addition `should_be_deployed` as a method name does not really offer any value/meaning here for me as a reader, as we pass method name as parameter.